admin/routes/index.js

```diff
@@ -41,60 +39,7 @@ router.get('/profile', isAuthenticated(), (req, res) => getProfile(req, res))
 /* Administration page */
 router.get('/administrator', isAuthenticated(config.ROLE_TEST_DEVELOPER), (req, res) => getAdministration(req, res))
 /* Health check */
-async function getPing (req, res) {
```

Comment:
We seem to have lost some functionality here.`getPing()` should restore commitId and build number now.

Reply:
@jon-shipley Odd... Code restored.